lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -308,6 +311,37 @@ def add_test_targets
             end
           end
 
+          # Adds the test app host targets for the library to the Pods project with the
+          # appropriate build configurations.
+          #
+          # @param  [Array<PBXNativeTarget>] test_native_targets
+          #         the test native targets that have been created to use as a lookup when linking the app host to.
+          #
+          # @return [Array<PBXNativeTarget>] the app host targets created.
+          #
+          def add_test_app_host_targets(test_native_targets)
+            target.test_spec_consumers.select(&:requires_app_host?).group_by(&:test_type).map do |test_type, test_spec_consumers|
+              platform = target.platform
+              name = "AppHost-#{target.label}-#{test_type.capitalize}-Tests"
```

Comment:
@segiddins I went with `target.label` which is already scoped by the platform/suffix.

This is now much better and much more symmetric to other names we use.

Reply:
👍